<launch>
<node pkg="depth_box_filter" type="depth_box_filter_node" name="depth_box_filter" output="screen">
    <!-- topics -->
    <remap from="image" to="/camera/depth/image_rect_raw"/>
    <remap from="camera_info" to="/camera/depth/camera_info"/>
    <param name="image_transport" type="str" value="compressedDepth" />

    <param name="base_frame" type="str" value="base" />

    <!-- planes in base frame -->
    <!-- parameter format: (nx,ny,nz,px,py,pz) -->
    <rosparam param="planes">
    [
        [1.0, 0.0, 0.0, 1.0, 0.0, 0.0],
        [0.0, 0.0, -1.0, 0.0, 0.0, 0.05]
    ]
    </rosparam>
</node>
</launch>
